### ISSUE TITLE
Filesystem on iOS deletes the files in the Documents directory and External Storage directory when deleting the app (documentation says files in these locations should persist)

### ISSUE THREAD

**Issue by NTass02** [ionic-team/capacitor-plugins#1679]
### Plugin(s)

Capacitor Filesystem

### Capacitor Version

```
Latest Dependencies:

  @capacitor/cli: 5.1.1
  @capacitor/core: 5.1.1
  @capacitor/android: 5.1.1
  @capacitor/ios: 5.1.1

Installed Dependencies:

  @capacitor/cli: 5.0.5
  @capacitor/core: 5.0.5
  @capacitor/android: 5.0.5
  @capacitor/ios: 5.0.5
```

### Code Reproduction

```
const options: WriteFileOptions = {
    path: 'test.pdf',
    data: data,
    directory: Directory.Documents,
}

return Filesystem.writeFile(options)
```

### Other Technical Details

Using Filesystem plugin with Ionic

**Comment by jcesarmobile** [comment]
The notes about files being deleted or not are all for Android, on iOS all files are deleted when the app is deleted, but we should make it more clear.

The docs doesn't really mention that they are persisted, on the other types they mention they are not, so people might assume they are persisted.

**Comment by ghenry22** [comment]
Could just reword it to say persistent for the life of the app install.

Yoh would have to put files outside the app owned folders somewhere (like the downloads area that the files app can access) in order for them to be persisted beyond the app uninstall.

**Comment by smollweide** [comment]
Hi all,
It is perfectly OK if the data in the Directory.Documents is deleted when the app is deleted on iOS. Especially because iOS informs the user that their data will be deleted. 

However, we have found out that the data is also deleted when the app is updated via the Apple FlightStore. Which I would then classify as critical. Can we assume that this does not happen with an update via the regular App Store?
